packages/react/rollup.config.js

```diff
@@ -0,0 +1,42 @@
+// rollup.config.js
+import { defineConfig } from 'rollup';
+import typescript from '@rollup/plugin-typescript';
+import { terser } from 'rollup-plugin-terser';
+import styles from 'rollup-plugin-styles';
+
+const config = defineConfig([
+  // CJS config
+  {
+    input: ['src/index.tsx', 'src/internal.tsx', 'src/legacy.tsx'],
+    output: {
+      dir: 'dist',
+      format: 'cjs',
+      sourcemap: true,
+    },
+    plugins: [typescript({ declarationDir: 'dist/types' }), terser()],
```

Comment:
What does terser do?

Reply:
terser is a code compressor. Using it to minify library size

Reply:
👍